packages/react/src/select/item/SelectItem.test.tsx

```diff
@@ -117,6 +117,29 @@ describe('<Select.Item />', () => {
     });
   });
 
+  it('should focus disabled items', async () => {
+    await render(
+      <Select.Root open>
+        <Select.Trigger data-testid="trigger">
+          <Select.Value data-testid="value" />
+        </Select.Trigger>
+        <Select.Portal>
+          <Select.Positioner>
+            <Select.Popup>
+              <Select.Item value="two" disabled>
+                two
+              </Select.Item>
+            </Select.Popup>
+          </Select.Positioner>
+        </Select.Portal>
+      </Select.Root>,
+    );
+
+    const item = screen.getByText('two');
+    act(() => item.focus());
+    expect(item).toHaveFocus();
+  });
```

Comment:
`Select` should now follow `Menu` with `useListNavigation`'s `disabledIndices`  prop